HRHN/View/UI/Cell/SettingCell.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LinkTableViewCell.swift
+//  HRHN
+//
+//  Created by Chanhee Jeong on 2022/12/28.
+//
+
+import UIKit
+
+class SettingCell: UITableViewCell {
+    static let identifier = "LinkTableViewCell"
+    
+    
+    override func updateConfiguration(using state: UICellConfigurationState) {
+        super.updateConfiguration(using: state)
+        tintColor = .tintColor
+    }
+    
+    func configureCell(with item: SettingItem){
+        var configuration = defaultContentConfiguration() // valuetype
+        configuration.text = item.text
+        configuration.secondaryText = item.secondaryText
+        configuration.secondaryTextProperties.color = .secondaryLabel
+        configuration.image = UIImage(systemName: item.imageName)
+        configuration.imageProperties.tintColor = item.type == .defaultItem ? .settingIconFill : .point
+        contentConfiguration = configuration
+        selectionStyle = .none
+        
+        switch item.type {
+        case .alertTime:
+            let timePicker = UIDatePicker()
+            timePicker.datePickerMode = .time
+            timePicker.tintColor = .point
+            timePicker.setValue(UIColor.point, forKeyPath: "textColor")
+            timePicker.addTarget(self,
+                                 action:#selector(timePickerdidChange(_:)),
+                                 for: .valueChanged)
+            accessoryView = timePicker
+        case .alertToggle:
+            let switchView = UISwitch(frame: .zero)
+            switchView.setOn(false, animated: true)
+            switchView.tag = 1
+            switchView.onTintColor = .point
+            switchView.addTarget(self, action: #selector(switchDidChange(_:)), for: .valueChanged)
+            accessoryView = switchView
+        case .defaultItem:
+            return
+        }
```

Comment:
<img width="412" alt="스크린샷 2022-12-29 오전 12 43 56" src="https://user-images.githubusercontent.com/75792767/209837040-951e09ab-d5c0-4653-b6a9-01c38d9e329e.png">
혹시 chevron이 없어진 이유가 있나요? 뭔가 아쉬워서,,, 한줄만 추가하면 되는데

```suggestion
        case .defaultItem:
            accessoryType = .disclosureIndicator
        }
```

Reply:
어...? 언제부터있었죠? 😅 
추가했습니다 🥹
f0ea578aeef4e1e416b6f351f22baf09a63161d5